### PR TITLE
fix(new-trace-replay) Remove duplicate vitals from web vitals tab

### PR DESCRIPTION
**Pull request by Abdkhan14**
We were collecting the vitals from the incrementally loaded tree twice during the append. `collectMeasurements` collects the vitals from the tree being appended and adds it to the root `tree.vitals`, which is the map used to render the content in the Web vitals tab in the trace drawer. 

Added transaction to vital headers to make them easier to distinguish.

Before:
Screenshot 2024-10-04 at 12 12 53 PM

After: 
Screenshot 2024-10-04 at 12 17 01 PM

### CODE REVIEW

File: static/app/views/performance/newTraceDetails/traceDrawer/tabs/traceVitals.tsx [hunk 1]
```diff
@@ -45,7 +46,7 @@ export function TraceVitals(props: TraceVitalsProps) {
                 </Tooltip>
                 <div>
                   <div>{t('transaction')}</div>
-                  <TraceDrawerComponents.TitleOp text={op} />
+                  <TraceDrawerComponents.TitleOp text={op + ' - ' + transaction} />
```
Comment (c298lee) [inline static/app/views/performance/newTraceDetails/traceDrawer/tabs/traceVitals.tsx:49]: what happens if there's no transaction?

Comment (Abdkhan14) [reply static/app/views/performance/newTraceDetails/traceDrawer/tabs/traceVitals.tsx:49]: There always will be, the typing is just a bit screwed

Comment (c298lee) [reply static/app/views/performance/newTraceDetails/traceDrawer/tabs/traceVitals.tsx:49]: oh, so `node.value.transaction` will always have a value?

Comment (Abdkhan14) [reply static/app/views/performance/newTraceDetails/traceDrawer/tabs/traceVitals.tsx]: I pushed a ternary just to be safe

**Review by c298lee — APPROVED** [review]

**Pull request merged**